File: 2405.1/comchk/c/ClassComment.cpp
#include <iostream>

using namespace std;

/**
 * 파일명 Main
 * @author 김지환
 * @version 1.0
 * @bad comment
 * @since 2021-01-19
 */
class People {
private:
    int age;
	string name;
	string address;

public:
	string getName() {
        return this->name;
    }
    void setName(string _name) {
        this->name = _name;
    }
};

File: 2405.1/comchk/java/ClassComment.java
package com.sparrow;

import java.lang.String;

/**
 * People 클래스
 * @author 김지환
 * @version 1.0
 * @bad comment
 * @since 2021-01-19
 */
public class People {
    private int age;
    private String name;
	private String address;

    public String getName() {
        return this.name;
    }
    public void setName(String name) {
        this.name = name;
    }
}